src/Assets/Standard Assets/Bugsnag/Bugsnag.cs

```diff
@@ -174,61 +174,57 @@ public class NativeBugsnag {
 
         public static void SetAutoNotify(bool autoNotify) {
             if (!CheckRegistration()) return;
-            if (autoNotify) {
-                Bugsnag.CallStatic ("enableExceptionHandler");
-            } else {
-                Bugsnag.CallStatic ("disableExceptionHandler");
-            }
+            BugsnagUnity.CallStatic ("setAutoNotify", autoNotify);
         }
 
         public static void SetContext(string context) {
             if (!CheckRegistration()) return;
-            Bugsnag.CallStatic ("setContext", context);
+            BugsnagUnity.CallStatic ("setContext", context);
```

Comment:
I may be misunderstanding how this is configured but in other places in this file we are calling `Bugsnag` directly rather than through `NativeBugsnag` e.g. [setContext](https://github.com/bugsnag/bugsnag-unity/blob/d8dc980b82a6006e5554f8a652b85f9076c7b42a/src/Assets/Standard%20Assets/Bugsnag/Bugsnag.cs#L478).

Should we only be configuring Bugsnag through the wrapper?

Reply:
It is quite confusing and I've spent quite a bit of time trying to pick through the flow of everything.

It looks like all of the methods on `Bugsnag` pass through to `NativeBugsnag` after performing some validation on the parameters. 

For Android the flow should go `Bugsnag` -> `NativeBugsnag` -> `BugsnagUnity` (the java class). It gets more confusing though, as in the implementation of `NativeBugsnag` for Android there is a static field named `Bugsnag` which is the java class for the Android client. I believe that this could be removed to reduce the confusion.